Source/Forms/Gen5/IDs5.cpp

```diff
@@ -158,20 +159,24 @@ void IDs5::search()
     connect(ui->pushButtonCancel, &QPushButton::clicked, [searcher] { searcher->cancelSearch(); });
 
     auto *timer = new QTimer();
-    connect(timer, &QTimer::timeout, [=] {
-        model->addItems(searcher->getResults());
-        ui->progressBar->setValue(searcher->getProgress());
-    });
+    connect(timer, &QTimer::timeout,
+            [=]
+            {
+                model->addItems(searcher->getResults());
+                ui->progressBar->setValue(searcher->getProgress());
+            });
     connect(thread, &QThread::finished, timer, &QTimer::stop);
     connect(thread, &QThread::finished, timer, &QTimer::deleteLater);
-    connect(timer, &QTimer::destroyed, [=] {
-        ui->pushButtonSearch->setEnabled(true);
-        ui->pushButtonFind->setEnabled(true);
-        ui->pushButtonCancel->setEnabled(false);
-        model->addItems(searcher->getResults());
-        ui->progressBar->setValue(searcher->getProgress());
-        delete searcher;
-    });
+    connect(timer, &QTimer::destroyed,
+            [=]
+            {
+                ui->pushButtonSearch->setEnabled(true);
+                ui->pushButtonFind->setEnabled(true);
+                ui->pushButtonCancel->setEnabled(false);
+                model->addItems(searcher->getResults());
+                ui->progressBar->setValue(searcher->getProgress());
+                delete searcher;
+            });
```

Comment:
Please use the .clang-format file in the repo. When I save it does not format to this

Reply:
LambdaBodyIndentation: OuterScope

Reply:
This should be formatted correctly now